recipes/wmctrl/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "wmctrl" %}
+{% set version = "0.3" %}
+{% set sha256 = "d806f65ac1554366b6e31d29d7be2e8893996c0acbb2824bbf2b1f49cf628a13" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
```

Comment:
It looks like this package does not support Python 3 due to the use of the deprecated `commands` stdlib package. Please add under `source`:

```
   skip: True   # [py3k]
```